website/cue/reference/releases/0.33.1.cue

```diff
@@ -0,0 +1,70 @@
+package metadata
+
+releases: "0.33.1": {
+	date:     "2023-10-30"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		This patch release contains fixes for regressions in 0.33.0 and fixes an issues with the Debian release artifacts.
+
+		**Note:** Please see the release notes for [`v0.33.0`](/releases/0.33.0/) for additional changes if upgrading from
+		`v0.32.X`. In particular, the upgrade guide for breaking changes.
```

Comment:
```suggestion
		`v0.32.X`. In particular, see the upgrade guide for breaking changes.
```

Reply:
Thanks! I opened https://github.com/vectordotdev/vector/pull/18987 to follow up on this.